site/src/pages/ManagementSettingsPage/OrganizationProvisionersPageView.stories.tsx

```diff
@@ -1,29 +1,49 @@
 import type { Meta, StoryObj } from "@storybook/react";
-import { MockProvisioner, MockUserProvisioner } from "testHelpers/entities";
+import {
+	MockBuildInfo,
+	MockProvisioner,
+	MockProvisioner2,
+	MockProvisionerWithTags,
+	MockUserProvisioner,
+} from "testHelpers/entities";
 import { OrganizationProvisionersPageView } from "./OrganizationProvisionersPageView";
 
 const meta: Meta<typeof OrganizationProvisionersPageView> = {
 	title: "pages/OrganizationProvisionersPage",
 	component: OrganizationProvisionersPageView,
+	args: {
+		buildInfo: MockBuildInfo,
+	},
 };
 
 export default meta;
 type Story = StoryObj<typeof OrganizationProvisionersPageView>;
 
 export const Provisioners: Story = {
 	args: {
-		provisioners: [
-			MockProvisioner,
-			MockUserProvisioner,
-			{
-				...MockProvisioner,
-				tags: {
-					...MockProvisioner.tags,
-					都市: "ユタ",
-					きっぷ: "yes",
-					ちいさい: "no",
-				},
-			},
-		],
+		provisioners: {
+			builtin: [MockProvisioner, MockProvisioner2],
+			psk: [MockProvisioner, MockUserProvisioner, MockProvisionerWithTags],
+			userAuth: [],
+			keys: new Map([
+				[
+					"ケイラ",
+					[
+						{
+							...MockProvisioner,
+							tags: {
+								...MockProvisioner.tags,
+								都市: "ユタ",
+								きっぷ: "yes",
+								ちいさい: "no",
+							},
+							warnings: [
+								{ code: "EUNKNOWN", message: "私は日本語が話せません" },
```

Comment:
Should there be a story showing the warning?
(I'd also remove the `私は` topic particle. It's not broken Japanese, but it will sound weird to native speakers)

Reply:
the warnings machinery works differently now than I imagined it would, so that'll get cleaned up in one of the following PRs